stock_production_lot_active/models/stock_production_lot.py

```diff
@@ -1,10 +1,18 @@
 # Copyright 2020 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class StockProductionLot(models.Model):
     _inherit = "stock.production.lot"
 
     active = fields.Boolean(string="Active", default=True)
+
+    # To avoid allowing duplicate lot/company/name combinations when there is another
```

Comment:
Put this as docstring of the method, not at comment at this level.

Reply:
done that and added a general description of the function as the first line